mod/settings.php

```diff
@@ -1106,40 +1106,40 @@ function settings_content(App $a)
 		$profile_in_dir = '<input type="hidden" name="profile_in_directory" value="1" />';
 	} else {
 		$profile_in_dir = replace_macros($opt_tpl, [
-			'$field' => ['profile_in_directory', L10n::t('Publish your default profile in your local site directory?'), $profile['publish'], L10n::t("Your profile may be visible in public."), [L10n::t('No'), L10n::t('Yes')]]
+			'$field' => ['profile_in_directory', L10n::t('Publish your default profile in your local site directory?'), $profile['publish'], L10n::t('If you enable this option your profile will be published in the global friendica directories (e.g. <a href="%s">%s</a>). Your profile may be visible in public.', Config::get('system', 'directory'), Config::get('system', 'directory')), [L10n::t('No'), L10n::t('Yes')]]
```

Comment:
`L10n::t('Your profile will be published in the global friendica directories (e.g. <a href="%1%s">%1%s</a>). Your profile details will be publicly visible.', Config::get('system', 'directory'))`

Reply:
I tried using only one  parameter for the replacement, but the magic does not work.

Reply:
Funny though, it does not work either. `sprintf` demands more arguments.